static/app/views/replays/detail/network/networkTableCell.tsx

```diff
@@ -121,7 +121,7 @@ const NetworkTableCell = forwardRef<HTMLDivElement, Props>(
       ),
       () => (
         <Cell {...columnProps}>
-          <Text>{statusCode ? statusCode : EMPTY_CELL}</Text>
```

Comment:
Can we make `EMPTY_CELL` = `--` or something similar as a placeholder? The white space makes it look empty and broken

Reply:
yes.

and for reference here are some other examples where we see those gaps:

<img width="701" alt="SCR-20230505-jzzw" src="https://user-images.githubusercontent.com/187460/236533320-f8d36381-d58d-4339-a71f-d946835ab472.png">
<img width="701" alt="SCR-20230505-kabl" src="https://user-images.githubusercontent.com/187460/236533324-5d94408a-243d-43f0-991b-55588d69272c.png">

Reply:
Updated version looks like:

<img width="700" alt="SCR-20230505-khft" src="https://user-images.githubusercontent.com/187460/236537349-16465b58-4578-4f2a-8547-40853004dcb0.png">

Reply:
IMO looks a lot better